### PR TITLE
[eclipse/xtext#1152] Java 9 - Added Automatic-Module-Name header

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

**Comment by szarnekow** [comment]
Anything missing here?

**Comment by kthoms** [comment]
No, just the approval.

File: org.eclipse.xtend.examples/projects/xtend-annotation-examples-client/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -9,4 +9,5 @@ Require-Bundle: org.eclipse.xtend.lib,
  org.eclipse.xtext.xbase.lib;bundle-version="2.14.0",
  xtend-annotation-examples;bundle-version="2.4.3"
 Bundle-Vendor: Eclipse Xtext
+Automatic-Module-Name: xtend-annotation-examples-client
```
Comment (szarnekow) [inline org.eclipse.xtend.examples/projects/xtend-annotation-examples-client/META-INF/MANIFEST.MF]: Would it make sense to use another module name here?

Comment (kthoms) [reply org.eclipse.xtend.examples/projects/xtend-annotation-examples-client/META-INF/MANIFEST.MF]: The convention is to use Bundle-SymbolicName
See https://dev.eclipse.org/mhonarc/lists/cross-project-issues-dev/msg14888.html

**Pull request merged**